model/integration_action.go

```diff
@@ -47,8 +47,16 @@ type PostAction struct {
 	// DataSource indicates the data source for the select action. If left
 	// empty, the select is populated from Options. Other supported values
 	// are "users" and "channels".
-	DataSource string               `json:"data_source,omitempty"`
-	Options    []*PostActionOptions `json:"options,omitempty"`
+	DataSource string `json:"data_source,omitempty"`
+
+	// Options contains either the buttons that will be displayed on the post
+	// or the values listed in a select dropdowon on the post.
+	Options []*PostActionOptions `json:"options,omitempty"`
+
+	// DefaultOption contains the option, if any, that will appear as the
+	// default selection in a select box. It has no effect when used with
+	// other types of actions.
+	DefaultOption string `json:"default_option,omitempty"`
```

Comment:
Now that I think of it, it might make more sense if this is called `DefaultValue` since this is actually the `value` of the default entry in `options` with how I have this written. Let me know if you think it makes sense to change that

Reply:
No I like DefaultOption because it clearly refers back to the previous setting.